std/string.d

```diff
@@ -6404,6 +6404,14 @@ S wrap(S)(S s, in size_t columns = 80, S firstindent = null,
     return result;
 }
 
+///
+@safe pure unittest
+{
+    assert(wrap("a short string", 7) == "a short\nstring\n");
+    assert(wrap("a short string", 7, "\t") == "\ta\nshort\nstring\n");
+    assert(wrap("a short string", 7, "\t", "    ") == "\ta\n    short\n    string\n");
```

Comment:
that is absolutely my point. If I have to look at the source what good is the example?